src/ValidationModels/penaltyValidation.js

```diff
@@ -1,14 +1,16 @@
 import Joi from 'joi';
 
+const regNoValidation = Joi.string().min(1).max(10)
```

Comment:
Does the app allow input of spaces for reg no? Or does it strip them out before submission?

Reply:
It strips them out. We previously had that validation only within `vehicleSchema` but not for `VehicleRegistration`.